lib/vanagon/platform/rpm/eos.rb

```diff
@@ -0,0 +1,83 @@
+class Vanagon
+  class Platform
+    class RPM
+      class EOS < Vanagon::Platform::RPM
+        # The specific bits used to generate an EOS package for a given project
+        #
+        # @param project [Vanagon::Project] project to build an EOS package of
+        # @return [Array] list of commands required to build the EOS package
+        # for the given project from an rpm or a swix
+        def generate_package(project)
+          # If nothing is passed in as platform type, default to building a swix package
+          if project.platform.package_type.nil? || project.platform.package_type.empty?
+            return generate_swix_package(project)
+          else
```

Comment:
this strange format here is for when we eventually specify multiple package types. If we have an array of different package types to build, then we can iterate over that. If it's an empty array, or not even set, then we just build one package, the default. Otherwise, we can iterate over the array. I was going to do that, but then I realized there are issues with the package_name method that I didn't want to solve.